### ISSUE TITLE
Wiley/Fish and Fisheries 

### ISSUE THREAD

**Issue by mark02139** [CDL-Dryad/dryad-product-roadmap#2446]
**Name to appear on membership page:** 
n/a

**Payment plan:**
Quarterly, volume-based attributed to Wiley

**Journals included:** 
Wiley has just informed us that they would like to add Fish and Fisheries (journal) to their agreement.
This needs to be included in Wiley's quarterly DPC invoicing

**Email contacts to receive submission notices: (note whether they receive all notices, or different lists for different journals)** 

**Email contacts to receive publication notices: (note whether they receive all notices, or different lists for different journals)** 

**ORCIDs of users who need access to admin dashboard: (they must log in to Dryad to set up their account before we can give them access, note whether each user administers a subset of journals)**

**Email contacts to discuss integration of journal metadata with Dryad:**

Development tasks:
- [ ] add member name to membership page
- [ ] update payment settings
- [ ] update journal notification settings
- [ ] add journal administrators
- [ ] start integration process (track on new ticket)
@ryscher @ahamelers @sdiggs @jleighherzog

**Comment by ryscher** [comment]
Fish and Fisheries is now on the Wiley payment plan.
